presentation/presentation.go

```diff
@@ -26,6 +28,15 @@ type uiContext struct {
 	prevReport   analysis.Report
 	cumulative   bool
 	paused       bool
+	useGui       bool
```

Comment:
If true, does this mean "make use of another GUI?" Suggest `enableTermboxGui` or similar.

Reply:
This was the existing useGui option passed through.  If true it uses termbox same as before...can rename it.